src/ConfigWidgets/SymbolsDialogTest.cpp

```diff
@@ -61,6 +63,10 @@ TEST_F(SymbolsDialogTest, ConstructEmpty) {
   auto* list_widget = dialog.findChild<QListWidget*>("listWidget");
   ASSERT_NE(list_widget, nullptr);
   EXPECT_EQ(list_widget->count(), 0);
+
+  auto* module_headline_label = dialog.findChild<QLabel*>("moduleHeadlineLabel");
+  ASSERT_NE(module_headline_label, nullptr);
+  EXPECT_FALSE(module_headline_label->isVisible());
```

Comment:
Maybe adding also an expect for the module to have text?

Reply:
This is a test where the SymbolsDialog is constructed without a module (Menubar > Settings > Symbol Locations). Hence there is no module and the moduleHeadlineLabel is hidden